pkg/service/manifest/manifest.go

```diff
@@ -163,3 +164,136 @@ func (s Service) DeleteManifest(request DeleteManifestRequest) error {
 
 	return nil
 }
+
+// TODO: (Neal) Add entire validation framework in place of these validation checks
```

Comment:
can create an issue and tag it here plz

Reply:
https://github.com/TBD54566975/ssi-service/issues/95